src/includes/performance/distributed-tracing/native.mdx

```diff
@@ -0,0 +1,45 @@
+To obtain trace headers from a transaction, use `sentry_transaction_iter_headers()`. 
+For a span, use `sentry_transaction_iter_headers()`. Pass the returned value to 
+the downstream service. If communication happens over HTTP, it is recommended to 
+attach all headers to the outgoing HTTP request.
+
+Continuing a trace from an upstream service requires using `sentry_transaction_context_update_from_header()`.
+Before starting a transaction, pass its transaction context into the aforementioned
+function accompanied with the `sentry-trace` header. The transaction started with
+the transaction context will contain everything needed to continue the trace.
+
+To obtain headers from a transaction so it can be continued from a downstream 
+service, define a function which merges the headers into some aggregate object. 
+The function is to be used in `sentry_transaction_iter_headers` as a callback.
+The following example uses `sentry_value_t` as the aggregate object:
+```c
+static void
+copy_headers_to(const char *key, const char *value, void *userdata)
+{
+    sentry_value_t *headers
+        = (sentry_value_t *)userdata;
+    sentry_value_set_by_key(*headers, key, value);
+}
+
+int main(int argc, char **argv) {
+  ...
+	// transaction to continue off of
+	sentry_transaction_context_t *tx_ctx
+        = sentry_transaction_context_new("honk", NULL);
+    sentry_transaction_t *tx = sentry_transaction_start(tx_ctx);
+
+	sentry_value_t *headers = sentry_value_new_object();
+	sentry_transaction_iter_headers(tx, copy_headers_to, (void *)headers);
+}
```

Comment:
@native, probably most importantly @Swatinem: i'll probably need a sanity check on the code and the example on this file.